wire-ios-utilities/Source/Public/AccentColor/AccentColor.swift

```diff
@@ -16,14 +16,19 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-@objc(ZMAccentColor)
 public enum AccentColor: Int16, CaseIterable, Hashable {
-    case undefined
-    case strongBlue
-    case strongLimeGreen
-    case brightYellow
-    case vividRed
-    case brightOrange
-    case softPink
-    case violet
+    case blue = 1
+    case green
+    // yellow used to be defined here
+    case red = 4
+    case amber
+    case turquoise
+    case purple
+}
+
+// MARK: - Default and random value
+
+extension AccentColor {
+    public static var `default`: Self { .blue }
+    public static var random: Self! { allCases.randomElement() }
```

Comment:
Shouldn't the force cast not be inside?
```suggestion
    public static var random: Self { allCases.randomElement()! }
```

Reply:
Actually I like it this way because of the following reason:
It's implicitly force-unwrapping when used like a non-optional property:
```
let randomAccentColor: AccentColor = .random
```
However, in unit tests you can test that it's not nil:
```
XCTAssertNotNil(AccentColor.random)
```
That way the tests don't crash if the implementation is wrong and nil is returned.

Reply:
I see your points, fine for me in this case. Often I tend more to have a bit aggressive behaviour and crash right away to see the problem (maybe the unit test gets removed).